src/map/MapRoutePath.js

```diff
@@ -79,17 +81,18 @@ const MapRoutePath = ({ name, positions, coordinates }) => {
     if (!coordinates) {
       coordinates = positions.map((item) => [item.longitude, item.latitude]);
     }
-    map.getSource(id)?.setData({
-      type: 'Feature',
-      geometry: {
-        type: 'LineString',
-        coordinates,
-      },
-      properties: {
-        name,
-        color: reportColor,
-      },
-    });
+    const maxSpeed = positions.map((p) => p.speed).reduce((a, b) => Math.max(a, b), -Infinity);
+    const features = [];
+    for (let i = 0; i < positions.length - 1; i += 1) {
+      const p1 = positions[i];
+      const p2 = positions[i + 1];
+      features.push(lineString(
```

Comment:
What does `lineString` function do?

Reply:
it's a helper from turf, creates a lineString feature

Reply:
I thought it's a trivial thing to create a line. What does the result look like?

Reply:
It is, it's just a helper, it creates the whole structure, including the properties (like it was done before manually):

{
      type: 'Feature',
      geometry: {
        type: 'LineString',
        coordinates: [[....]]
      },
      properties: {
        color: '...',
      },
};

Reply:
I prefer to keep it as JSON instead of using helper.

Reply:
ok, lineString is gone.